nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/main/java/org/apache/nifi/provenance/lucene/DocsReader.java

```diff
@@ -47,9 +46,6 @@
 public class DocsReader {
     private final Logger logger = LoggerFactory.getLogger(DocsReader.class);
 
-    public DocsReader(final List<File> storageDirectories) {
-    }
-
```

Comment:
Should consult the versioning guide to see if it addresses how to handle this.  We need to be very explicit about which parts of the codebase are public and designed for extension versus those which the community should be free to evolve .  if the guide doesn't cover this case let's solve it.